### PR TITLE
Update FAQs

### PR DESCRIPTION
**Pull request by ananyo2012**
Updated the FAQs

### CODE REVIEW

**Comment by ananyo2012** [comment]
@pythonindia/pycon-2018-team Please review

File: faq.html [hunk 1]
```diff
@@ -139,31 +137,17 @@ <h5><b>Q. Will the Internet and WiFi setup be good enough for a large crowd? Sho
...
         <div class="pycon-faq">
           <h5><b>Q. What all does the Conference ticket (Oct 6 and Oct 7, 2018) include? </b></h5>
           <ul class="faq-answer">
-                <li> - Full conference pass for two days.</li>
-                <li>- Breakfast, Lunch and Hi-Tea</li>
-                <li>- PyCon India T-shirt which will be distributed on Oct 6, 2018.</li>
-          </ul>
-        </div> 
-        <div class="pycon-faq">
-          <h5><b>Q. My talk is not selected for conference, can I opt for OpenSpace? </b></h5>
```
Comment (realslimshanky) [inline faq.html]: I guess we can comment this and the next questions/segment out as we'll be needing these in future instead of removing it completely.

Comment (ananyo2012) [reply faq.html]: Done.

**Comment by ananyo2012** [comment]
@realslimshanky Thanks for reviewing. Merging this.

**Pull request merged**